Bin/Expandflexentities.php

```diff
@@ -0,0 +1,197 @@
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the Hoa nor the names of its contributors may be
+ *       used to endorse or promote products derived from this software without
+ *       specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS AND CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+namespace Hoa\Devtools\Bin;
+
+use Hoa\Consistency\Consistency;
+use Hoa\Console;
+use Hoa\File;
+
+/**
+ * Class \Hoa\Devtools\Bin\ExpandFlexEntities
+ *
+ * This command resolve the class_alias function in destination to IDE.
+ *
+ * @copyright  Copyright © 2007-2017 Hoa community
+ * @license    New BSD License
+ */
+class Expandflexentities extends Console\Dispatcher\Kit
+{
+    /**
+     * Options description.
+     *
+     * @var array
+     */
+    protected $options = [
+        ['output', Console\GetOption::REQUIRED_ARGUMENT, 'o'],
+        ['dry-run', Console\GetOption::NO_ARGUMENT, 'd'],
+        ['verbose', Console\GetOption::NO_ARGUMENT, 'v'],
+        ['help', Console\GetOption::NO_ARGUMENT, 'h'],
+        ['help', Console\GetOption::NO_ARGUMENT, '?']
+    ];
+
+    /**
+     * The entry method.
+     *
+     * @return  void
+     */
+    public function main()
+    {
+        $dryRun  = false;
+        $output  = null;
+        $verbose = false;
+
+        while (false !== $c = $this->getOption($v)) {
+            switch ($c) {
+                case 'd':
+                    $dryRun = true;
+
+                    break;
+
+                case 'o':
+                    $output = $v;
+
+                    break;
+
+                case 'V':
+                    $verbose = true;
+
+                    break;
+
+                case 'h':
+                case '?':
+                    return $this->usage();
+
+                case '__ambiguous':
+                    $this->resolveOptionAmbiguity($v);
+
+                    break;
+            }
+        }
+
+        if (null === $output) {
```

Comment:
Can be removed if `$output` is set to `php://stdout` by default. Would be logic after all.